packages/form-js-viewer/src/render/components/Sanitizer.js

```diff
@@ -45,6 +45,7 @@ const ALLOWED_ATTRIBUTES = [
 
 const ALLOWED_URI_PATTERN = /^(?:(?:(?:f|ht)tps?|mailto|tel|callto|cid|xmpp):|[^a-z]|[a-z+.\-]+(?:[^a-z+.\-:]|$))/i; // eslint-disable-line no-useless-escape
 const ALLOWED_IMAGE_SRC_PATTERN = /^(https?|data):.*/i; // eslint-disable-line no-useless-escape
+const ALLOWED_IFRAME_SRC_PATTERN = /^(https?):\/\/*/i; // eslint-disable-line no-useless-escape
```

Comment:
@volodymyr-melnykc @christian-konrad @vsgoulart @Skaiir We could also update this pattern to only allow for `https`, which is recommended anyways.

Wdyt?

Reply:
Yeah, I think thatś a good idea

Reply:
Let's also be explicit to the user that we require a `https` URL. 
@pinussilvestrus I updated the copy for the tooltip. Can you please update it in the codebase?

<img width="472" alt="image" src="https://github.com/bpmn-io/form-js/assets/143816833/7e295ac8-039d-4fb2-9127-6595e3d8365b">

Reply:
Updated it accordingly (preview still building) 👍